examples/react/css/src/variables.css

```diff
@@ -7,6 +7,8 @@
   --ais-primary-color-alpha: 1;
   --ais-muted-color-rgb: 127, 137, 164;
   --ais-muted-color-alpha: 0.6;
+  --ais-color-white-rgb: 255, 255, 255;
+  --ais-color-white-alpha: 1;
```

Comment:
Should we use a more semantic naming? If customers want to customize it, it's weird to set `--ais-color-white-rgb` to a different color.